src/stories/components/RecentPosts/styles.ts

```diff
@@ -0,0 +1,22 @@
+import { Typography, styled } from '@mui/material';
+import { spacing } from 'theme/spacing';
+export const Wrapper = styled('div')`
+  color: ${({ theme }) => theme.palette.primary.main};
+`;
+
+export const Title = styled(Typography)`
+  color: ${({ theme }) => theme.palette.grey['700']};
```

Comment:
Sugiro trocar por outra propriedade do tema que pega as cores de texto, como por exemplo: `theme.palette.text.primary`. Avalie qual cor fica melhor para essa situação.
Você pode consultar as cores padrões do MUI [aqui](https://mui.com/system/palette/)

Sempre que for colocar cor em textos use cores do tema pegando `theme.palette.text[variant]` por exemplo. 
Sei que no nosso código não está dessa forma em todos os lugares, mas devemos ficar atentos para sempre manter esse padrão

Irei fazer somente um comentário referente a isso, mas peço que altere em todos os lugares que a cor do texto não está definida com a cor de tema